frontend/src/app/home/home.component.ts

```diff
@@ -14,14 +15,12 @@ export class HomeComponent implements OnInit {
     is_settings_displayed: boolean;
     widgets: {"name": string, "displayed": boolean}[]
     associations: any[];
-    status_message_html: string;
 
-    constructor(private _apiService: ApiService) {
+    constructor(private _apiService: ApiService, private notifier: NotifierService) {
```

Comment:
_apiService mais notifier : pourquoi pas la même convention (sur l'underscore comme sur le suffixe "Service") ?

Reply:
The leading underscore looks like bad practice (https://angular.io/guide/styleguide#style-03-04), we should follow the same convention everywhere though

Reply:
Agreed, I would write apiService and notifierService

Reply:
👍